src/amo/components/SearchResult/index.js

```diff
@@ -55,15 +55,39 @@ export class SearchResultBase extends React.Component<InternalProps> {
 
     let imageURL = iconURL;
 
+    const imageAtts = {};
+
     if (isTheme) {
+      // Since only newly created static themes will have more than one preview
+      // we will set up a fallback for now.
+      const previewFallback = addon && addon.previews && addon.previews[0];
+      const previewSearch = addon && addon.previews && addon.previews[1];
+      const previewImage = previewSearch || previewFallback;
```

Comment:
Just out of interest, why do we prefer `previews[1]` over `previews[0]`?

Reply:
Assuming `previews` is directly from the api, the first item is the header preview, and the second item is the list (search) preview that's longer, and that's ideally the one you want to use.  (Also, it's a list, so if there wasn't a `previews[0]` there wouldn't be a `previews[1]`)

Reply:
or... I don't really understand what the code does I'm commenting on 🤣 (did I get it right @rebmullin ?)

Reply:
@bobsilverberg, yeh as @eviljeff mentions the first array item is for the header and the 2nd is for search/shelves/grid..

@eviljeff , I guess I was going off this comment [here](https://github.com/mozilla/addons-server/issues/8412#issuecomment-400050380). So I temporarily set up a fallback since I noticed on older static themes, the previews didn't come up.  I think I added a comment for this too. Anyway, I thought it was better to set it up this way..?

Reply:
@rebmullin yeah.  It _shouldn't_ happen on stage and prod, but it's a sensible fallback.

Reply:
ok cool : ) yeh it was a temporary set up really for dev :|